src/Simulation/EntryPointDriver/Microsoft.Quantum.EntryPointDriver.csproj

```diff
@@ -16,7 +15,11 @@
 
```

Comment:
```
<PackageReference Include="Microsoft.Quantum.Compiler" Version="0.10.2004.1717-alpha" />
```

I think that this is bringing in a lot of DLLs from the compiler that weren't there before when you run `dotnet publish` on a standalone application (see microsoft/qsharp-compiler#432). This is probably not what we want.

The only reason EntryPointDriver has this dependency is to get the constants defined in ReservedKeywords for the names of simulators and command line arguments. A few alternatives:

1. Read the constants at compile time (CsharpGeneration naturally has a dependency on the compiler) and embed them in the generated C# entry point file, and pass those into the driver so it can use them instead. This requires explicitly listing out all the constants the driver uses and making a new class to hold them, but their values will still come from the compiler.
2. Create a separate package in the compiler repository for ReservedKeywords and reference that instead of the full compiler.
3. Hard-code the strings instead of using the defined constants.

Thoughts? @bettinaheim @anpaz-msft

Reply:
I think it is slightly different than EntryPointInfo, since that holds operation-specific data, while this class would have global constants that only have one value throughout the program, which makes a class feel weird. But yes I agree that it's still probably the best option.

Reply:
+1 for option 1, too.